sql_metadb/derived_tables/runlist.txt

```diff
@@ -6,6 +6,7 @@ agreements_subscription_agreement_org_ext.sql
 finance_transaction_invoices.sql
 finance_transaction_purchase_order.sql
 holdings_ext.sql
+holdings_statistical_codes
```

Comment:
The suffix `.sql` is missing

Reply:
@adoerrer You may have accidentally approved this pull request before the error above was fixed.

Reply:
Yes, I've missed that. Thank you, for catching that!

Reply:
This is still not fixed.  I will add a commit to fix it.